plan/2.0 Expand Training.md

```diff
@@ -38,338 +16,519 @@
 
-- **M1**: Establish a job description for educational content developer.
-- **M2**: Hire education content developer
-- **M3**: select channels content will be delivered in
-- **M4**: create roadmap for content creation and publication
-- **M5**: create content for channels
-- **M6**: Develop in-person educator-led variations of training materials (e.g., with hands-on lab work) & go to conferences to encourage training
-- **M7**: Develop SKF labs for new training materials
+| Milestone | SKF Ops | Contractor(s) | Time Estimate |  Year  |
+| :-------: | :-----: | :-----------: | :-----------: | :----: |
+|    M1     |   ✅    |       -       |     80hrs     | Year 1 |
+|    M2     |   ✅    |       -       |  40hrs(each)  | Year 1 |
 
-### Time & resource estimate
+> **SKF OPS TEAM - TOTAL**: 160 hours
 
-- Y1 - Hire Education Content Developer program manager- (1 person x $ 300K): $ 300K
-- Y1 - XX hours for roadmap development and content creation
+---
 
-## 2.6.2: Expand Secure Development thought leadership through blogs, webinars, podcasts
+## 2.7 Goal: Alternate delivery streams for content
 
-### Explanation: Many traditional channels are available to augment traditional classes such as pdocasts, blogs, and webinars. These are generally inexpensive and readily-accessioble tools to amplify the awareness of key concepts and techniques
+### Explanation: Sponsor existing or create alternate delivery streams for educational content
 
 ### Key Steps/Milestones
 
-- **M1**: Write 12 blogs supporting SIG & WG efforts
-- **M2**: Create and deliver 2 secure development webinars
-- **M3**: Create "Secure from the (open) Source" podcast and produce 12 episodes
+- **M1**: Establish a job description for an educational content developer.
 
-### Time & resource estimate
+  - [ ] Create a job description for an educational content developer
+  - [ ] Get feedback on the job description
+  - [ ] post opening and interview candidates
+  - [ ] onboard secdev contributors
 
-- Y1 - $35K to equip and produce podcast including graphics and music
-- Y2+ - $6k annual for ongoing poscast expenses
-- 60 volunteer hours to write 12 blogs - annual
-- 24 OSSF support hours to facilitate editing and publication of 12 blogs - annual
-- 48 volunteer hours to create and record 2 webinars - annual
-- 24 OSSF support hours to facilitate the recording, marketing, and publishing of 2 webinars - annual
-- 60 volunteer hours to record, produce, and publish 12 podcasts (audio and video) - annual
-- 12 OSSF support hours to facilitate podcast marketing and publication - annual
+- **M2**: Create a calendar of conferences where our content would be useful to train
 
-## 2.6.3: Expand Secure Development education with content to "training camps" and other groups
+  - [ ] Identify potential conferences where we want to present the materials and platform
+  - [ ] Contact conferences and apply for CFP / CFT
+  - [ ] Create a calendar of selected conferences
```

Comment:
Recommend also adding in here the creation of a tutorial format for said conferences.
the tutorial format should be structured such that a 2-3 qualified individuals can execute an in person or virtual tutorial that is different enough from the self-paced labs and content.  they should be tailored such that no tutorial is identical to a previous one delivered.